pkgs/applications/blockchains/chia/default.nix

```diff
@@ -12,7 +12,8 @@ python3Packages.buildPythonApplication rec {
     owner = "Chia-Network";
     repo = "chia-blockchain";
     rev = version;
-    sha256 = "sha256-yS0/Fy2dj8VIbwv2J9sehP0VN0f/YDxu1k9WkaeEz8M=";
+    fetchSubmodules = true;
```

Comment:
I am fine with vendoring some typescript/js https://github.com/Chia-Network/chia-blockchain-gui but vendoring a ca is not fine for me https://github.com/Chia-Network/mozilla-ca. We should use the nix one.

Reply:
> I am fine with vendoring some typescript/js https://github.com/Chia-Network/chia-blockchain-gui but vendoring a ca is not fine for me https://github.com/Chia-Network/mozilla-ca. We should use the nix one.

I am confused as to what derivation the canonical Mozilla CA bundle lives in. please tell me.

[`nix-locate cacert.pem`](https://gist.github.com/lourkeur/bd389ba7922bb952b342cbaf55992c87)

Reply:
just use `cacert` i think

Reply:
Do we still need the submodules if we are using `cacert`?

Reply:
good question: yes we do, there are still one useful file in that module besides the CA bundle. (currently an empty `__init__.py`, but that could change) Besides, there's also the gui bits which may become useful later.